### PR TITLE
support for python 3

### PR DESCRIPTION
**Pull request by ltkum**
I kept xrange in the code rather than already switchint to range (which is also supported in python2), because performance-wise, xrange is better in python2 than range (the comportment of range in python3 has been altered to be the same as xrange). This is ugly, but should be temporary.

I also put little TODOS to explain what is left to do to finish the transition. It's not much.

### CODE REVIEW

**Comment by ltkum** [comment]
I made a small PR with no content to try the double build (with this branch as a base): both tests passed (it's now closed : https://github.com/geoadmin/service-alti/pull/73). In my opinion, we can merge this.

**Comment by procrastinatio** [comment]
I also launch both build manually. It builds, without error. Have you disabled liting?

File: alti/lib/profile_helpers.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import math
+import six
```
Comment (procrastinatio) [inline alti/lib/profile_helpers.py:2]: Didn't check. Is `six` in `requirements.txt`?

File: Makefile [hunk 2]
```diff
@@ -52,16 +52,44 @@ RED := $(shell tput setaf 1)
 GREEN := $(shell tput setaf 2)
 
 # Versions
-PYTHON_VERSION := $(shell python --version 2>&1 | cut -d ' ' -f 2 | cut -d '.' -f 1,2)
+ifndef USE_PYTHON3
+		override USE_PYTHON3 = 0
+endif
+
+ifeq ($(USE_PYTHON3), 1)
+		PYTHON_VERSION := 3.8.3
```
Comment (procrastinatio) [inline Makefile:60]: Goal is to dockerize, right? It won't work on our debian setting.

**Pull request merged**